### DISCUSSION TITLE
Aeon Labs Home Energy Meter Gen5 1-C60 not updating values automatically

### DISCUSSION THREAD

**Discussion by adn-coder** [zwave-js/zwave-js#7459] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

When I paired a Home Energy Meter Gen5 - ZW095C160A the measured values are not updated. I have to "manually" refresh the Meterv4 parameters.

I also see that in "GROUPS"  menu nothing is mapped : 
![image](https://github.com/user-attachments/assets/4cfc6e36-fe94-4e95-b090-10fff28b9a6f)

If I manually set it like that it work well : 
![image](https://github.com/user-attachments/assets/40127606-df64-4d15-91b5-c678ff646aae)

My question is the following : is it possible to configure the JSON file related to this device to automatically set the association parameter as i need to ?

Please find attached my JSON file : 
[zw095.json](https://github.com/user-attachments/files/18006556/zw095.json)

### Software versions

Z-Wave JS v12.9.1

Home Energy Meter SW v1.37

### Device information

Manufacturer: AEOTEC
Model name: Home Energy Meter ZW095-1C60

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
@adn-coder we really need a driver log of a re-interview to diagnose this, like the issue template asks for.
